src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -39,12 +39,14 @@ if NOT [%ERRORLEVEL%]==[0] (
     echo Warning: An error occurred while configuration https port.
   )
 
+REM
 REM Start the self hosted WCF Test Service
-call %~dp0..\..\..\..\bin\Wcf\tools\SelfHostedWcfService\SelfHostedWcfService.exe %*
+REM
+call %~dp0..\..\..\..\bin\Wcf\tools\SelfHostedWcfService\SelfHostedWcfService.exe
 set __EXITCODE=%ERRORLEVEL%
 
+:Cleanup
+call %~dp0WCFCustomCleanup.cmd WCFServiceCleanupNeeded
```

Comment:
What is the purpose of the argument passed to WCFCustomCleanup.cmd?  As far as I can tell, the environment variable is always set to 'true' and the CMD does not look at arguments (plus you are not passing "true" but instead the string "WcfServiceCleanupNeeded" as %1).  I recommend deleting this argument for now.